File: java/Payment.java
class Payment {
    
    Integer id;
    Integer monto;

    public Payment(Integer id, Integer monto){
        this.id = id;
        this.monto = monto;
    }

    void printDataCar(){
        System.out.println("id: " + id + " monto: " + monto);
    }
    
}

File: php/Payment.php
<?php
class Payment
{
    public $id;
    public $monto;

    public function __construct($id, $monto){
        
        $this->id = $id;
        $this->monto = $monto;

    }

    public function printDataPayment(){
        
        echo  " id: $this->id monto: ".$this->monto;

    }

}
?>